docs/architecture.md

```diff
@@ -0,0 +1,93 @@
+# Project Architecture
```

Comment:
@Harish-osmosys  Update readme so users can navigate to these documents from ReadME. Users looking at readme won't know about these in current state

Reply:
updated in https://github.com/OsmosysSoftware/supply-chain-blockchain/pull/5/commits/f0a9eea9645c3517951ce350477fd46392f1bff9